src/App.vue

```diff
@@ -58,9 +43,45 @@
         max-size="70"
         min-size="10"
       >
-        <SpecEditor v-model="code" />
+        <SpecToolbar class="editor-toolbar">
+          <template #left>
+            <strong>
+              API specification
+            </strong>
+          </template>
+          <template #right>
+            <KButton
+              appearance="secondary"
+              size="small"
+              @click="dropzoneClick()"
```

Comment:
nit: i think you just need this?
```suggestion
              @click="dropzoneClick"
```